go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -503,14 +503,20 @@ func (c *Controller) syncNodeInfos(nodeInfos []nodeInfo) {
 		}
 	}
 
-	// Sync the nodeIP template values to the DB.
-	nodeIPTemplates := []TemplateMap{
-		c.nodeIPv4Templates.AsTemplateMap(),
-		c.nodeIPv6Templates.AsTemplateMap(),
-	}
-	if err := svcCreateOrUpdateTemplateVar(c.nbClient, nodeIPTemplates); err != nil {
-		klog.Errorf("Could not sync node IP templates")
-		return
+	c.startupDoneLock.RLock()
+	defer c.startupDoneLock.RUnlock()
```

Comment:
do we need to keep hold of the lock while processing `svcCreateOrUpdateTemplateVar` below?